docs/UserGuide.md

```diff
@@ -142,6 +142,21 @@ Tags:`
 
 ![result for 'add n/John p/80101010 e/johndoe@gmail.com a/Punggol Central Blk 444 #15-32 820123 nk/Brennan nkp/82020202'](images/addUi.png)
 
+You can also add a client's contacts with multiple tags.
+
+Examples:
+* `add n/John p/80101010 e/johndoe@gmail.com a/Punggol Central Blk 444 #15-32 820123 nk/Brennan nkp/82020202 t/80yo t/grumpy`
+
+Successful Output: `New person added: John; 
+Phone: 80101010; 
+Email: johndoe@gmail.com; 
+Address: Punggol Central Blk 444 #15-32 820123; 
+Next-of-kin Name: Brennan; 
+Next-of-kin Phone: 82020202; 
+Appointment: No Appointment made!; 
+Financial Plans: ; 
+Tags: [80yo][grumpy]`
+
```

Comment:
If we include an example for tags, would it be inconsistent if we do not include a case for financial tags too? I think it may be better in the example usage just have one adding tag and financial tag, then maybe have a inline block with tips saying u can add multiple by duplicating prefix

Reply:
To add on should also change the addUI.png